mobly/controllers/android_device.py

```diff
@@ -366,7 +366,7 @@ def __init__(self, serial="", host_port=None, device_port=8080):
         self.adb_logcat_file_path = None
         self.adb = adb.AdbProxy(serial)
         self.fastboot = fastboot.FastbootProxy(serial)
-        if not self.is_bootloader:
+        if not self.is_bootloader and self.is_rootable:
```

Comment:
Seems like this PR also mixes in the adb root change?
I'm guessing you didn't start it on a clean branch?